### ISSUE TITLE
Ethereum Genesis Sate .json included in the build

### ISSUE THREAD

**Issue by haadcode** [ipfs/js-ipfs#744]
When building orbit-db examples, which use js-ipfs, I noticed there's a big .json file included in the build:
```
 [425] ./~/ethereum-common/genesisState.json 628 kB {0} [built]
```

Which comes from:
```
  └─┬ ipfs@0.21.7
    └─┬ ipld-resolver@0.4.3
      └─┬ ipld-eth-block@2.2.1
        └─┬ ethereumjs-block@1.4.1
          ├── ethereum-common@0.0.18
          └─┬ ethereumjs-tx@1.2.4
            └── ethereum-common@0.0.17
```

Not sure if this is needed but we should make sure to verify if it is and if not, remove it. This would give us a huge reduction in the minified build footprint.

**Comment by diasdavid** [comment]
@kumavis can you check this?

**Comment by kumavis** [comment]
we do not need it, i will see it removed

**Comment by kumavis** [comment]
cc @wanderer

**Comment by kumavis** [comment]
fixed in `ethereumjs-tx@1.2.5`
ethereumjs-block pending publish by @wanderer 
https://github.com/ethereumjs/ethereumjs-block/issues/25

**Comment by haadcode** [comment]
Thank you @kumavis for such a quick action! 🎉 ❤️

**Comment by kumavis** [comment]
@haadcode fixed in  `ethereumjs-block@1.5.0`

**Comment by diasdavid** [comment]
sweet! closing this one then :) thank you @kumavis
